### DISCUSSION TITLE
How to change anything (handwriting, sketches etc) in saved .xopp files?

### DISCUSSION THREAD

**Discussion by dooomkopf** [xournalpp/xournalpp#5375] [General]
i'm not able to change anything, if file is saved once and re-openend. is this supposed to be so? i already worked with xournal, here everything is adjustable, rearrangable, erasable etc.
any suggestions?

**Comment by rolandlo** [comment]
No, this is not supposed to be like that. Can you share a `.xopp`-file that you have saved?

**Reply by dooomkopf** [reply]
![notEdible](https://github.com/xournalpp/xournalpp/assets/22260307/2739e938-4db3-4e30-bd90-385fdcca6994)

you know, it's not a jpeg...

**Reply by dooomkopf** [reply]
i have v1.2.1, update right now...

**Comment by rolandlo** [comment]
The problem is not the version, the problem is that you have added a (mostly empty) extra layer and are editing the second layer instead of the first. Maybe you hit `<Ctrl>L` without noticing?!
You can view the different layers in the sidebar (layer preview). You can also switch between the layer that is currently edited there.

**Reply by dooomkopf** [reply]
sakra nochamol, wär ich nie drauf gekommen. that's it. problem solved! thanks for quick help.
